uni/lib/view/course_unit_info/widgets/course_unit_sheet.dart

```diff
@@ -0,0 +1,107 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_widget_from_html/flutter_widget_from_html.dart';
+import 'package:html/dom.dart' as dom;
```

Comment:
Why import like this?

Reply:
Notice I'm using `com.Element` later on in the file. It's the easier way to disambiguate from Flutter's view `Element`.